ViewModel/ImpactAreaScenario/IASElementSet.cs

```diff
@@ -159,24 +156,22 @@ private ObservableCollection<ImpactAreaRowItem> GetStudyImpactAreaRowItems()
 
         private List<SpecificIASResultVM> GetResults()
         {
-            
             List<SpecificIASResultVM> results = new List<SpecificIASResultVM>();
-
             //this is kind of messy. Quite a bit of code to get the name of the impact area from the impact area id.
+            //todo: get a list of result objects
+            List<metrics.Results> iasResults = new List<metrics.Results>();
```

Comment:
Namespacing

Reply:
I tried to get rid of "metrics" but "Results" is also a namespace and VS can't seem to figure it out without the full path.